lib/plugins/inspec-compliance/lib/inspec-compliance/api.rb

```diff
@@ -24,19 +24,7 @@ class API
       # the username of the account is used that is logged in
       def self.profiles(config, profile_filter = nil) # rubocop:disable Metrics/PerceivedComplexity, Metrics/CyclomaticComplexity, Metrics/AbcSize, Metrics/MethodLength
         owner = config["owner"] || config["user"]
-
-        # Chef Compliance
-        if is_compliance_server?(config)
-          url = "#{config["server"]}/user/compliance"
-        # Chef Automate2
-        elsif is_automate2_server?(config)
-          url = "#{config["server"]}/compliance/profiles/search"
-        # Chef Automate
-        elsif is_automate_server?(config)
-          url = "#{config["server"]}/profiles/#{owner}"
-        else
-          raise ServerConfigurationMissing
-        end
```

Comment:
This isn't quite the same - if the server config is missing, that error won't be caught. I think we still need to check for `is_automate2_server?`.

Reply:
Yes! Nice catch. Will preserve this check.